README.rst

```diff
@@ -1013,6 +1013,19 @@ The default configuration is as follows:
         'max_allowed_backoff': 15,
     }
 
+``OPENWISP_NOTIFICATIONS_EMAIL_BATCH_INTERVAL``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This seems longer than it should

Reply:
I assumed that it would be okay since we already use this setting [`OPENWISP_NOTIFICATIONS_POPULATE_PREFERENCES_ON_MIGRATE`](https://github.com/openwisp/openwisp-notifications/#openwisp-notifications-populate-preferences-on-migrate)

Reply:
Federico was pointing to the heading underline. The length of the underline should be equal to the length of the text. 

```suggestion
``OPENWISP_NOTIFICATIONS_EMAIL_BATCH_INTERVAL``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```